examples/cn0565/eit_dynamic_bp.py

```diff
@@ -0,0 +1,62 @@
+# coding: utf-8
+""" demo code for CN0565 EIT back-projection """
+# Copyright (c) Benyuan Liu. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
```

Comment:
What is the origin of this file?

Reply:
Updated example and used SPDX. This file was created by us.